news/4.31/jdt.html

```diff
@@ -64,6 +64,18 @@ <h2>JUnit</h2>
     <h2>Java Editor </h2>
     </td>
   </tr>
+
+  <tr id="added-make-method-static"> <!-- https://github.com/eclipse-jdt/eclipse.jdt.ui/pull/689 -->
+    <td class="title"><a href="#added-make-method-static">New 'Make Static' for methods</a></td>
```

Comment:
I might suggest "New Make Static Refactoring" as the title.

Reply:
> I might suggest "New Make Static Refactoring" as the title.

I understand, I like your proposal and edited my N&N accordingly